src/Games/NexusMods.Games.Larian/BaldursGate3/BaldursGate3.cs

```diff
@@ -0,0 +1,63 @@
+﻿using NexusMods.Abstractions.GameLocators;
+using NexusMods.Abstractions.GameLocators.GameCapabilities;
+using NexusMods.Abstractions.GameLocators.Stores.GOG;
+using NexusMods.Abstractions.GameLocators.Stores.Steam;
+using NexusMods.Abstractions.Games;
+using NexusMods.Abstractions.Games.DTO;
+using NexusMods.Abstractions.IO;
+using NexusMods.Abstractions.IO.StreamFactories;
+using NexusMods.Abstractions.Loadouts.Synchronizers;
+using NexusMods.Paths;
+
+namespace NexusMods.Games.Larian.BaldursGate3;
+
+public class BaldursGate3 : AGame, ISteamGame, IGogGame
+{
+    public override string Name => "Baldur's Gate 3";
+    
+    public IEnumerable<uint> SteamIds => [1086940u];
+    public IEnumerable<long> GogIds => [1456460669];
+
+    public static GameDomain GameDomain => GameDomain.From("baldursgate3");
+    public override GameDomain Domain => GameDomain;
+    
+    public BaldursGate3(IServiceProvider provider) : base(provider)
+    {
+    }
+    public override GamePath GetPrimaryFile(GameStore store)
+    {
+        // TODO: Check linux and osx paths, linux should be the same, osx is unknown
```

Comment:
You can use SteamDB for this. It has the file content list for a depot.

Reply:
I fixed the executable path, but other paths are also broken, specifically the Data path, since it should have `Contents` prepended to it. I'm thinking of just returning `location.Path.Join("Contents")` as the game folder for MacOS, all the files are inside that folder anyways. It would normalize the rest of the differences a lot.